app/src/app/layout/services/sidebar.service.ts

```diff
@@ -48,9 +45,9 @@ export class SidebarService {
   }
 
   /**
-   * Reload the category tree.
+   * Load Category Tree {@link CmdbCategoryTree}.
    */
-  public reload() {
+  public loadCategoryTree() {
```

Comment:
I find reload was the more fitting name, because the tree is reloaded when it is executed again.

Reply:
Called it 'loadCategoryTree, because I needed a function to initialize the values. 
But since the initialisation does the same as 'reload' and I didn't want to have two methods doing the same thing or one method calling the other, I thought this was the better alternative. 
But here I am open for improvements. Suggest something.